demosplan/DemosPlanCoreBundle/DoctrineMigrations/2024/03/Version20240313120508.php

```diff
@@ -46,6 +47,13 @@ public function up(Schema $schema): void
     public function down(Schema $schema): void
     {
         $this->abortIfNotMysql();
+
```

Comment:
I think these changes are not intended for this pr? Does the same as the up migration :thinking:

Reply:
solved by merge target branch